clinica/iotools/utils/data_handling.py

```diff
@@ -142,6 +142,9 @@ def create_merge_file(
                                 for key, value in json_dict.items():
                                     new_col_name = f"{modality}_{key}"
                                     scans_dict.update({new_col_name: value})
+                    scans_dict = {
+                        str(key): str(value) for key, value in scans_dict.items()
+                    }
```

Comment:
Why the explicit conversion?  Instead, I think `pd.DataFrame(..., dtype="string")` should work.

Reply:
This is kind of a 'hot-fix' because the merge was breaking from the modifications in [PR 213](https://github.com/aramis-lab/clinica/pull/213). 

This part of the code needs another major refactoring.